website/docs/feature-flag-tutorials/python/examples.md

```diff
@@ -0,0 +1,508 @@
+}
+
+response = requests.request("PUT", url, headers=headers, data=payload)
+
+print(response.text)
+```
+
+> Note:
+> If you already have a gradual rollout strategy for your flag, use a PUT request to update it.
+> If you’re creating a new strategy, use a POST request.
+
+Check out our [API docs on updating flag strategies](/reference/api/unleash/update-feature-strategy) to learn more.
+
+Read our documentation for more context on [strategy constraint configurations](/reference/strategy-constraints) and use cases.
+
+
+## Server-side A/B Testing in Python
+
+
+A/B testing is a common way for teams to test out how users interact with two or more versions of a new feature that is released. Server-side A/B testing can help with making infrastructure improvements and comparing different versions of server-side methods. At Unleash, we call these flag [variants](/reference/feature-toggle-variants).
+
+When a feature flag is enabled, we can expose a particular version of a feature to select user bases. From there, we can use the variants to view the performance metrics in Unleash and see which is more efficient. 
+
+In the context of our [Python tutorial](/feature-flag-tutorials/python), let’s say we want to test a new variation of the flag for deleting surveys. In this new variation, we want the UI to remain the same–a delete button that removes a survey–but server-side, we’d like to store the deleted surveys in a new database model so they can restored if a user decides to.
+
+While we won’t implement all the functionality required to save and restore deleted surveys in this example, we can walk through how to set up variants and where the server-side changes would take place in the Python app.
+
+In Unleash, navigate to the feature flag Variants tab, then ‘Add variant’.
+
+![Add a variant in the development environment.](/img/python-ex-add-variant.png)
+
+In the form, add 2 variants: 
+
+`store_deleted_surveys` <br /> `permanently_delete_surveys`
+
+
+![Two variants can be configured to Unleash and save it to your strategy.](/img/python-ex-add-variants.png)
+
+Alternatively, you can also use a `PATCH` request in Python using our API:
+
+```py
+import requests
+import json
+
+url = "<your-unleash-url>/api/admin/projects/:projectId/features/:featureName/environments/:environment/variants"
+
+payload = json.dumps([
+ {
+    "op": "replace",
+    "path": "/1/name",
+    "value": "permanently_delete_surveys"
+  },
+  {
+    "op": "replace",
+    "path": "/0/name",
+    "value": "store_deleted_surveys"
+  }
+])
+headers = {
+  'Content-Type': 'application/json',
+  'Accept': 'application/json',
+  'Authorization': '<API_KEY_VALUE>'
+}
+
+response = requests.request("PATCH", url, headers=headers, data=payload)
+
+print(response.text)
+```
+
+Your Variant tab now has the new variants we just created with their respective metadata.
+
+![View the list of your variants per environment on the Variants page.](/img/python-ex-variants-added.png)
```

Comment:
Like I mentioned before, I think it would be better to use [strategy variants](https://docs.getunleash.io/reference/strategy-variants) here instead. I think they would be simpler for new users and a cool bonus is that their API looks nicer, since it's part of the strategy, so same endpoints as before.

![image](https://github.com/Unleash/unleash/assets/14320932/60b1ab7b-7adb-4087-bbc4-e74176cc90ef)

Reply:
Yes thank you for catching that! to your point, and when I spoke with @ivarconr recently, strategy variants are the way to go.